docs/Events/2024_05_24_geoseries_14.md

```diff
@@ -0,0 +1,25 @@
+# Geoseries EP 13 (Technical Session)
+
+### Title: Large Language Model (LLM) and its Geospatial Applications
+
+**Date and Time:** 2024-05-24 Friday, 7:00 PM (Nepali Time)
+
+**Coordinator and Moderator:** Tek Kshetri
+
+**Poster By:** Aayush Chand
+
+**Presentor:** Rohit Gautam
+
+**Affiliation:** Geospatial Developer, NAXA
+
+### Short Description
+
+Rohit Gautam specializes in geospatial software solutions integrating maps, location data, and advanced analytics. He has contributed to popular open-source mapping projects like Leafmap and created OsmGPT, a tool that uses AI to retrieve data from OpenStreetMap through plain language queries. He explores new applications for large language models and generative AI by combining mapping with advanced AI technologies.
+
+### Video Recording
+
+https://youtu.be/7bO8-1sL0hE?si=_Z0BS_AMPBiYmYgo
+
+### Poster
+
```

Comment:
@rowheat02, can you please share your slides? The better approach might be, upload it to the slideshare site and add the URL here.

Reply:
Thank you Rohit.